### PR TITLE
New execution module: statuspage

### PR DESCRIPTION
**Pull request by mirceaulinic**
### What does this PR do?

To handle easier requests to the StatusPage API: http://doers.statuspage.io/api/v1/

In the salt-ish style, the API and the page ID is taken from the config file if not sent on the CLI.

### CODE REVIEW

**Comment by cachedout** [comment]
Go Go Jenkins!

**Review by cachedout — CHANGES_REQUESTED** [review]
Pylint has some issues here: https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/5252/violations/file/salt/modules/statuspage.py/

**Comment by mirceaulinic** [comment]
Damn! Sorry about that @cachedout! I have pushed the fix.

File: salt/modules/statuspage.py [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+.. versionadded:: Nitrogen
+'''
+
+from __future__ import absolute_import
+
+# import python std lib
+import logging
+
+# import third party
+import requests
```
Comment (cachedout) [inline salt/modules/statuspage.py]: I think we should move this to a try/except inside the `__virtual__` function.

Comment (mirceaulinic) [inline salt/modules/statuspage.py]: Isn't `requests` one of the requirements? https://github.com/saltstack/salt/blob/develop/requirements/base.txt#L5

**Review by cachedout — APPROVED** [review]

**Pull request merged**